rxbindings/build.gradle

```diff
@@ -30,4 +30,5 @@ dependencies {
         // https://github.com/robolectric/robolectric/issues/5245
         exclude group: 'com.google.auto.service', module: 'auto-service'
     }
+    testImplementation project(path: ':aws-api')
```

Comment:
Let's just kill all the tests that use models, or change model to string. The rxbinding (even tests) should only know about the `core` module, not any plugin.

Reply:
I'm afraid this has a big impact in how tests were written initially. I would be nervous to make that change given the size of this PR.

In the future I assume we would be able to change this dependency into the shared module one, like we do in iOS.

Reply:
spoke offline and I was able to understand what Jameson suggested. This was fixed.